### PR TITLE
add PSClient in worker

### PR DESCRIPTION
**Pull request by QiJune**
We want to support both TensorFlow and PyTorch.  Both TensorFlow worker and PyTorch worker will pull parameters from PS and push gradients to PS. We should implement a class called `PSClient`, which provides common methods for both TensorFlow worker and PyTorch worker.

It's the first PR for migrating methods to `PSClient`. I will move more PS related methods in next PRs.

### CODE REVIEW

**Comment by skydoorkai** [comment]
We need to put all PS related calls to PSClient.
So this is the first PR to implement PSClient?

File: elasticdl/python/worker/ps_client.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        self._ps_num = len(self._ps_stubs)
+
+    def pull_embedding_vectors(self, layer_name, embedding_ids):
+        """
+        Pulls and returns embedding vectors ordered by the embedding ids.
+        Args:
+            layer_name: layer name
+            embedding_ids: a list of ids
+        Return:
+            embedding_vectors: a numpy ndarray
```
Comment (workingloong) [inline elasticdl/python/worker/ps_client.py]: a 2-D numpy array?

**Comment by QiJune** [comment]
@skydoorkai  Yes, it's the first PR. I will move more PS related methods in next PRs.

File: elasticdl/python/worker/ps_client.py (hunk 1 above)
Comment (QiJune) [reply elasticdl/python/worker/ps_client.py]: Done.

**Review by skydoorkai — APPROVED** [review]
